app/services/EnrolmentsStoreService.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import javax.inject.{Inject, Singleton}
+
+import com.google.inject.ImplementedBy
+import connectors.EnrolmentStoreConnector
+import models.{UserEnrolmentStatus, UserEnrolments, VatEnrolment, VatVarEnrolment}
+import org.joda.time.{DateTime, DateTimeZone}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class EnrolmentStoreServiceImpl @Inject()(connector: EnrolmentStoreConnector)(implicit val ec:ExecutionContext) extends EnrolmentsStoreService {
+
+  val daysBetweenExpectedArrivalAndExpiry = 23
```

Comment:
"Magic number". Shall we keep this in the conf file?

Reply:
I don't know, it seems very cumbersome to put a value like that in the conf file when it's only used in one place and makes little sense out of context - I already had to give it a horrible name in context.